setup.cfg

```diff
@@ -5,3 +5,5 @@ exclude = *migrations*
 statistics=true
 application-import-names = incuna_auth
 import-order-style = google
+# https://github.com/public/flake8-import-order/issues/24
+ignore = I201
```

Comment:
I don't like this. It would be better tow downgrade the version of `flake8-import-order`.